packages/a/arc-icon-theme/package.yml

```diff
@@ -1,8 +1,9 @@
 name       : arc-icon-theme
 version    : '20161122'
-release    : 7
+release    : 8
 source     :
     - https://github.com/horst3180/arc-icon-theme/archive/20161122.tar.gz : 0b448dde34138b12ab2d5f224f001cf8203fff55e6b19b013b0461b4376641f9
+homepage   : https://github.com/horst3180/arc-icon-theme/
 license    : GPL-3.0
```

Comment:
Could you please use a [SPDX License](https://spdx.org/licenses/).
When the developers don't specify if it is `GPL-3.0-only` or `GPL-3.0-or-later` we prefer `GPL-3.0-or-later`.